mirage/route-handlers/me.js

```diff
@@ -98,7 +98,7 @@ export function register(server) {
     return { ok: true };
   });
 
-  server.get('/api/v1/me/crate_owner_invitations', function (schema) {
+  server.get('/api/private/crate_owner_invitations', function (schema) {
```

Comment:
it looks like several tests are failing now, which makes sense, because they are currently still using the old endpoint. in this first step I would only add the new mirage endpoint, but also keep the old one at the same time, until we've migrated the frontend code to use the new endpoint.

Reply:
Addressed on [7ecadfd](https://github.com/rust-lang/crates.io/pull/5397/commits/7ecadfd0f90d39732bf4a13d9092552ed7fa826b)